[2022-08-13 12:17:34,491] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.create_raw_data_dir manual__2022-08-13T12:17:33.304434+00:00 [queued]>
[2022-08-13 12:17:34,502] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.create_raw_data_dir manual__2022-08-13T12:17:33.304434+00:00 [queued]>
[2022-08-13 12:17:34,504] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 12:17:34,505] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-08-13 12:17:34,505] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 12:17:34,523] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): create_raw_data_dir> on 2022-08-13 12:17:33.304434+00:00
[2022-08-13 12:17:34,528] {standard_task_runner.py:52} INFO - Started process 3767 to run task
[2022-08-13 12:17:34,532] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_Raízen', 'create_raw_data_dir', 'manual__2022-08-13T12:17:33.304434+00:00', '--job-id', '343', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmph2t_2ng6', '--error-file', '/tmp/tmpp4y8evi2']
[2022-08-13 12:17:34,533] {standard_task_runner.py:80} INFO - Job 343: Subtask create_raw_data_dir
[2022-08-13 12:17:34,608] {task_command.py:369} INFO - Running <TaskInstance: ETL_Raízen.create_raw_data_dir manual__2022-08-13T12:17:33.304434+00:00 [running]> on host 4bbb34367c41
[2022-08-13 12:17:34,702] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_Raízen
AIRFLOW_CTX_TASK_ID=create_raw_data_dir
AIRFLOW_CTX_EXECUTION_DATE=2022-08-13T12:17:33.304434+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-13T12:17:33.304434+00:00
[2022-08-13 12:17:34,704] {ETL_Raizen.py:27} INFO - Dir already exists
[2022-08-13 12:17:34,704] {python.py:173} INFO - Done. Returned value was: None
[2022-08-13 12:17:34,719] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Raízen, task_id=create_raw_data_dir, execution_date=20220813T121733, start_date=20220813T121734, end_date=20220813T121734
[2022-08-13 12:17:34,746] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-13 12:17:34,793] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
